imageBuilder.go

```
@@ -9,32 +9,65 @@ import (
 
 var publishImage = flag.Bool("publishImage", false, "Push image to docker hub")
 var codeRunnerConfig = flag.String("codeRunnerConfig", "code-runner.yaml", "Code Runner configuration filename")
+var assignmentEnvConfig = flag.String("assignmentEnvConfig", "assignment-env.yaml", "Assignment Environment configuration filename")
 var dockerAuthConfig = flag.String("dockerAuthConfig", "docker-auth.yaml", "Docker hub authentication configuration filename")
 var dockerfileName = flag.String("dockerfile", "Dockerfile", "Dockerfile name")
 
 func main() {
+
 	flag.Parse()
 
 	log.Println("Creating Dockerfile...")
 
+	imgBuilder := builder.NewImageBuilder(*dockerAuthConfig, *dockerfileName)
+
+	err := generateCodeRunnerImage(imgBuilder)
+	if err == nil {
+		_ = generateAssignmentEnvImage(imgBuilder)
```

Why ignore the error?

---

Changed